packages/edit-site/src/components/header/close-area/index.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * WordPress dependencies
+ */
+import { createSlotFill } from '@wordpress/components';
+
+const { Fill: __experimentalSiteEditorCloseArea, Slot } = createSlotFill(
+	'__experimentalSiteEditorCloseArea'
+);
+
+__experimentalSiteEditorCloseArea.Slot = Slot;
+
+export default __experimentalSiteEditorCloseArea;
```

Comment:
I need to come up with a better name for this. :)

Reply:
Yeah thats a tough one... should we replace `Area` with `SlotFill` ? or maybe there is a better term someone can suggest.

Reply:
I don't think it's bad.

But, since this is the "home button", maybe `__experimentalSiteEditorHomeButton`?

Either one works IMO.

Reply:
Yes, I'd avoid referring to "close" or any implicit action and try to be a bit more semantic (something like `MainDashboardButton` perhaps).

Reply:
Renamed to `MainDashboardButton` in 555b399532aa0e2a02bbc07d24ce9c3f22d0d868.